tests/PerformanceTests/PerformanceTestService/EventSource/BenchmarkMsalDistributedTokenCacheAdapter.cs

```diff
@@ -40,8 +72,18 @@ protected override async Task RemoveKeyAsync(string cacheKey)
         /// (account or app).</returns>
         protected override async Task<byte[]> ReadCacheBytesAsync(string cacheKey)
         {
+            var stopwatch = Stopwatch.StartNew();
+            var bytes = await _distributedCache.GetAsync(cacheKey).ConfigureAwait(false);
+            stopwatch.Stop();
+
             MemoryCacheEventSource.Log.IncrementReadCount();
-            return await base.ReadCacheBytesAsync(cacheKey);
+            MemoryCacheEventSource.Log.AddReadDuration(stopwatch.Elapsed.TotalMilliseconds);
```

Comment:
Is it to measure the time of the the distributed store read?

Reply:
Yes.